src/main/java/com/johncnstn/auth/security/DomainUserDetailsService.java

```diff
@@ -20,11 +20,11 @@
     @Override
     @Transactional(readOnly = true)
     @SuppressWarnings("")
```

Comment:
Remove this @SuppressWarnings(""), please

Reply:
Done